src/lightning_app/cli/lightning_cli.py

```diff
@@ -29,10 +31,68 @@ def get_app_url(runtime_type: RuntimeType, *args) -> str:
 @click.group()
 @click.version_option(ver)
 def main():
-    register_all_external_components()
     pass
 
 
+@main.group()
+def show():
+    """Show info about resource."""
+    pass
+
+
+@show.command()
+@click.argument("app_name", required=False)
+@click.argument("components", nargs=-1, required=False)
+@click.option("-f", "--follow", required=False, is_flag=True)
+def logs(app_name: str, components: List[str], follow: bool) -> None:
+
+    client = LightningClient()
+    project = _get_project(client)
+
+    apps = {
+        app.name: app
+        for app in client.lightningapp_instance_service_list_lightningapp_instances(project.project_id).lightningapps
+    }
+
+    if not apps:
+        raise click.ClickException("Your app list is empty. Please run an application first.")
+
+    if not app_name:
+        raise click.ClickException(
+            f"You have not specified any LightningApp. Please select one of available: [{', '.join(apps.keys())}]"
+        )
+
+    if app_name not in apps:
+        raise click.ClickException(f"LightningApp '{app_name}' does not exist.")
```

Comment:
nit: show list of apps like above `Please select one of available: [{', '.join(apps.keys())}]`

Reply:
👍